src/libraries/Native/Unix/configure.cmake

```diff
@@ -805,9 +805,14 @@ check_type_size(
      BUILTIN_TYPES_ONLY)
 set(CMAKE_EXTRA_INCLUDE_FILES) # reset CMAKE_EXTRA_INCLUDE_FILES
 
-check_include_files(
-    "sys/types.h;sys/sysctl.h"
-    HAVE_SYS_SYSCTL_H)
+if (CLR_CMAKE_TARGET_LINUX)
```

Comment:
We have also HAVE_SYSCTL in src/coreclr/src/pal/src/configure.cmake  and src/coreclr/src/gc/unix/configure.cmake. It looks like it should get the same treatment. And we also have HAVE_SYS_SYSCTL_H there, which can be deleted from those scripts and the related config.h.in files as it is not used by coreclr PAL or gc anymore.